solutions/gold/cf-1181D.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: cf-1181D
+source: CF
+title: Irrigation
+author: Chongtian Ma
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/67727)
+
+## Explanation
+
+We can subtract $n$ from each of the queries, and answer them offline. Then, sort the cities by occurrences and queries by year.
```

Comment:
```suggestion
We can subtract $n$ from each query, answer them offline, and then sort the cities by occurrences and the queries by year.
```

Reply:
thats not the right order bro
you should sort before u answer queries offline

Reply:
in ur initial sentences it isn't clear, so I suggest you fix it